test/sequential/test-module-loading.js

```diff
@@ -127,10 +127,6 @@ assert.equal(require('../fixtures/registerExt2').custom, 'passed');
 assert.equal(require('../fixtures/foo').foo, 'ok',
              'require module with no extension');
 
-assert.throws(function() {
```

Comment:
Can you add a test to confirm that `require.paths` returns `undefined` or whatever the expected value is?